gui/app/controls/readings/IerDisplay.qml

```diff
@@ -6,5 +6,8 @@ import ".."
 ParameterDisplay {
     parameterName: qsTr("I:E")
     parameterUnit: qsTr("ratio")
-    parameterValue: GuiStateContainer.measured_ier.toFixed(2).toString()
+    parameterValue: {
+      var inv = 1.0 / GuiStateContainer.measured_ier
+      return "1:" + inv.toFixed(Number.isInteger(inv) ? 0 : 1)
```

Comment:
If inv is say 1.0001, presumably you want to output `1:1` not `1:1.0`?  Especially since inv is a measured value?  As is, it seems you'd almost never output without the decimal place.

Seems like maybe you want to round to 1 decimal place and then check if it's integral?

```
var inv = Number((1.0 / measured_ier).toFixed(1));
```

Not sure if you care how 0.5 is rounded.  JS's round() function rounds to +inf.  Whereas toFixed seems to round to nearest odd, which is totally weird but I guess is better than round to +inf.